tools/qsscheck.py

```diff
@@ -242,14 +242,14 @@ def get_global_names(mixxx_path):
             ext = os.path.splitext(fname)[1]
             if ext in (".h", ".cpp"):
                 fpath = os.path.join(root, fname)
-                with open(fpath, mode="r") as f:
+                with open(fpath, mode="r", encoding="utf-8") as f:
                     for line in f:
                         classnames.update(set(RE_CPP_CLASSNAME.findall(line)))
                         objectnames.update(set(RE_CPP_OBJNAME.findall(line)))
             elif ext == ".ui":
                 fpath = os.path.join(root, fname)
-                with open(fpath, mode="r") as f:
-                    objectnames.update(set(RE_UI_OBJNAME.findall(f.read())))
+                with open(fpath, mode="r", encoding="utf-8") as f:
+                objectnames.update(set(RE_UI_OBJNAME.findall(f.read())))
```

Comment:
pretty sure its broken without the indent (cuz python)
```suggestion
                with open(fpath, mode="r", encoding="utf-8") as f:
                    objectnames.update(set(RE_UI_OBJNAME.findall(f.read())))
```